libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -512,8 +512,56 @@
     }
   },
   "marketParticipant": {
-    "apiErrorCodes": {
-      "globalLocationNumberValidation": "'Aktørnummer' skal være et gyldigt GLN eller EIC nummer"
+    "market_participant": {
+      "error_fallback": "Computeren blev bedt om at udføre en handling, men gjorde det ikke. Årsagen var: {{ message }}",
+      "bad_argument": {
+        "missing_required_value": "Feltet '{{ param }}' kræver en værdi, men denne er ikke blevet angivet.",
+        "invalid_gln": "Værdien '{{ value.value }}' er ikke et gyldigt GLN- eller EIC-nummer.",
+        "invalid_length": "Indholdet af '{{ param }}' er for langt. Maksimal længde er {{ max }} tegn.",
```

Comment:
invalid_length can also be minimum length?

Reply:
Fixed.